src/common/__test__/addNew.test.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @jest-environment jsdom
+ */
+import localStorageArray from '../__mocks__/localStorage.js';
+import addNew from '../__mocks__/addNew.js';
+import deleteItem from '../__mocks__/deleteItem.js';
+
+const mockHtml = '<ul class="listContainer"></ul>';
+const listHTML = (obj) => {
+  const list = `<li class="toDoList" id="list-${obj.index}">
+  <input class="checkbox" type="checkbox" id="checkbox-${obj.index}" ${obj.complete ? 'checked' : ''}>
+  <input class="description" type="text" id="todo-${obj.index}" value="${obj.description}">
+  <button class="delete" type="button" id="button-${obj.index}"></button>
+  </li>`;
+  return list;
+};
+
+test('New Entry should save to localStorage', () => {
+  document.body.innerHTML = mockHtml;
+  const toDoContainer = document.querySelector('.listContainer');
+  const object = {};
+  object.description = 'Car Wash';
+  object.index = 1;
+  object.complete = false;
+  toDoContainer.innerHTML += listHTML(object);
+  expect(toDoContainer.children.length).toBe(1);
+  localStorageArray.push(object);
+  expect(localStorageArray.length).toBe(1);
+  const fetchList = toDoContainer.children[0];
+  const fetchText = fetchList.querySelector('.description');
+  expect(fetchText.value).toMatch(object.description);
```

Comment:
- [ ] Let's group your tests using the `description()` method as requested in project requirement.

Reply:
Added the same.